pkg/czid/local.go

```diff
@@ -31,10 +34,31 @@ func IsR2(path string) bool {
 	return r2Exp.MatchString(path)
 }
 
+func extractLaneNumber(path string) (int, error) {
+	match := sampleNameExp.FindString(path)
+	if len(match) < 5 {
+		log.Fatalf("path has no lane number %s", path)
+	}
+
+	n, err := strconv.Atoi(match[2:5])
+	if err != nil {
+		return n, fmt.Errorf("path has no lane number %s", path)
+	}
+	return n, nil
+}
+
+func sortByLaneNumber(arr *[]string) {
```

Comment:
If we only have lanes L001 through L008, could we then sort the array of strings as is without extracting the lane number?